java/arcs/android/common/resurrection/DbHelper.kt

```diff
@@ -44,10 +42,7 @@ class DbHelper(
 ) {
     init {
         // Pre-initialize parsers.
-        // TODO: Make a master storage key parsing initializer in arcs.core.storage.
-        RamDiskStorageKey.registerParser()
-        VolatileStorageKey.registerParser()
-        ReferenceModeStorageKey.registerParser()
+        DriverAndKeyConfigurator.configureKeyParsers()
```

Comment:
Why is this still needed?

Reply:
I suppose it's not really.. but it doesn't hurt I guess.